packages/office-ui-fabric-react/src/components/Sticky/Sticky.types.ts

```diff
@@ -29,6 +29,11 @@ export interface IStickyProps extends React.Props<Sticky> {
    * @defaultvalue true
    */
   isScrollSynced?: boolean;
+
+  /**
+   * Class name to apply to the sticky element if component is sticky & scroll position is non-zero.
+   */
+  onScrollStickyClassName?: string;
```

Comment:
Can you elaborate on why we need another prop?  Does `stickyClassName` not work for your scenario(s)?

Reply:
I have to apply a css class when component is sticky & scroll position is non zero

Reply:
I'm not sure I follow this.  Anytime you scroll, the scroll position will be non zero, so shouldn't `stickyClassName` work for your scenario then?